templates/app/assets/stylesheets/components/search/_search_bar.scss

```diff
@@ -34,3 +34,49 @@
     padding: 0 2rem 0 1rem;
   }
 }
+
```

Comment:
Isn't this CSS duplicated with what we have in the `_autocomplete_results.scss` file?

Reply:
lol, yes, I started there, then moved to a different file and forgot to remove it, thanks!!!